src/main/java/net/sf/jabref/logic/cleanup/Cleanups.java

```diff
@@ -18,17 +23,35 @@
 public class Cleanups {
 
     public static final FieldFormatterCleanups DEFAULT_SAVE_ACTIONS;
+    public static final FieldFormatterCleanups RECOMMEND_BIBTEX_ACTIONS;
+    public static final FieldFormatterCleanups RECOMMEND_BIBLATEX_ACTIONS;
     public static List<Formatter> availableFormatters;
 
+
     static {
         availableFormatters = new ArrayList<>();
         availableFormatters.addAll(Formatters.ALL);
 
         List<FieldFormatterCleanup> defaultFormatters = new ArrayList<>();
         defaultFormatters.add(new FieldFormatterCleanup(FieldName.PAGES, new NormalizePagesFormatter()));
+        defaultFormatters.add(new FieldFormatterCleanup(FieldName.DATE, new NormalizeDateFormatter()));
         defaultFormatters.add(new FieldFormatterCleanup(FieldName.MONTH, new NormalizeMonthFormatter()));
-        defaultFormatters.add(new FieldFormatterCleanup(FieldName.BOOKTITLE, new OrdinalsToSuperscriptFormatter()));
+        defaultFormatters.add(new FieldFormatterCleanup(FieldName.ABSTRACT_ALL_TEXT_FIELDS_FIELD, new OrdinalsToSuperscriptFormatter()));
         DEFAULT_SAVE_ACTIONS = new FieldFormatterCleanups(false, defaultFormatters);
+
+        List<FieldFormatterCleanup> recommendedBibTeXFormatters = new ArrayList<>(defaultFormatters);
+        recommendedBibTeXFormatters.add(new FieldFormatterCleanup(FieldName.TITLE, new HtmlToLatexFormatter()));
```

Comment:
Is there a reason to not run the HTML/Unicode -> Latex converter on all [text] fields? Similar question for the Latex/HTML -> Unicode for biblatex.